docs/interactions/structured-messages/apple-auth.md

```diff
@@ -88,8 +88,8 @@ The customer will receive a message including the body and a button that will op
 }
 ```
 
-## Example: Apple Business Chat (Apple Pay)
+## Example: Apple Messages for Business (Apple Pay)
 
-Nothing specifically unique as this is an Apple Business Chat specific structured message type.
+Nothing specifically unique as this is an Apple Messages for Business specific structured message type.
```

Comment:
![apple1](https://user-images.githubusercontent.com/1233351/159460447-ca2e4430-4f47-4031-aeca-ad09e6a609a7.png)

Reply:
btw on this one I replaced "Apple Pay" with "Apple Authentication", was probably a copy paste error from a long time ago